docs/introduction/distribution.rst

```diff
@@ -0,0 +1,10 @@
+What is the Distribution 2.0?
+===========================
+
+iGEM is designing and building a new DNA distribution entirely from the ground up with support from the FreeGenes Project and the iGEM Engineering Committee. The focus of this and future distributions is in improving the quality of the parts and devices within, through curation and new interlaboratory studies run by the Engineering Committee. We hope these new changes will help iGEM teams push all facets of the design-build-test cycle for their projects further.
+
+The new iGEM distribution will be manufactured and distributed by the FreeGenes Project, with all physical material provided under the BioBrick Foundation’s Unilateral OpenMTA. Anyone will be able to request and use the iGEM distribution! 
+
+This also marks the first time that iGEM is collaboratively designing the distribution. We’ve worked closely with the iGEM Engineering Committee on the contents of this year’s distribution. This has also led to collaborative building and testing of new software infrastructure to support this endeavor. All of the associated material within this distribution, from the contents to the software, is available through this github repository!
```

Comment:
*GitHub

Reply:
Following a revised version with some additional points,   Need to ensure accuracy.  Some further clarification required.
-------------------------

The iGEM Distribution was initiated to put a toolkit of genetic parts at the disposal of IGEM teams.  The goal of the program is to enable is to enable teams to devote more of their effort and attention to the design and invention of unique concepts and effective solutions and to reduce time spent searching for and developing the components on which achieving satisfactory results depends.    

The output of the distribution is a diverse range of elements such as plasmids, xxxx,  whose development is complex, involving rationalization of disparate naming conventions, translation between incompatible structures, modification of sequences 

What distinguishes this year’s distribution program from those of previous years and represents an important step forward is the scale of the collaborative effort undertaken to support the program and the introduction of a new software infrastructure designed to increase the efficiency of development. 

At the heart of the collaborative effort is the Engineering Committee whose members have devoted significant time and effort to ensuring the creation of useful and usable content.  However, the extent of the collaboration has not been limited to internal contributors, but encompasses external parties as well.   FreeGenes, Twist, and xxx are just a few of the organization that have contributed to and supported the distribution program. 

All of the material associated with this distribution, from the contents to the software, as well as supplemental information resides within the  GitHub repository (provide link here).